src/coreclr/src/tools/Common/Compiler/DependencyAnalysis/SettableBlobNode.cs

```diff
@@ -0,0 +1,60 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+
+using Internal.Text;
+using Internal.TypeSystem;
+
+namespace ILCompiler.DependencyAnalysis
+{
+    public class SettableBlobNode : ObjectNode, ISymbolDefinitionNode
+    {
+        private Utf8String _name;
+        private ObjectNodeSection _section;
+        private int _alignment;
+        private ObjectData _data;
+
+        public SettableBlobNode(Utf8String name, ObjectNodeSection section, int alignment)
```

Comment:
The alignment is actually coming from the `ObjectData`, so this parameter and field is unused.

Reply:
I removed alignment. Thank you!